pycobertura/reporters.py

```diff
@@ -443,3 +443,47 @@ def generate(self):
                     )
 
         return template.render(**render_kwargs)
+
+
+class GitHubAnnotationReporter(Reporter):
+    def __init__(
+        self,
+        cobertura,
+        ignore_regex=None,
+        annotation_level: str = "error",
+        title: str = "pycobertura",
+        message: str = "not covered",
+    ):
+        super().__init__(cobertura, ignore_regex)
+        self.annotation_level = annotation_level
+        self.title = title
+        self.message = message
+
+    def generate(self):
+        file_names = self.cobertura.files(ignore_regex=self.ignore_regex)
+        result_strs = []
+        for file_name in file_names:
+            for rang in rangify(self.cobertura.missed_lines(file_name)):
```

Comment:
Can we unpack the output of `rangify()`?

```suggestion
            for range_start, range_end in rangify(self.cobertura.missed_lines(file_name)):
```